src/content/docs/fr/guides/cms/buttercms.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: ButterCMS & Astro
+description: Ajouter du contenu à votre projet Astro en utilisant ButterCMS
+type: cms
+service: ButterCMS
+stub: false
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+ButterCMS est un CMS headless et un moteur de blog qui vous permet de publier du contenu structuré à utiliser dans votre projet.
```

Comment:
"à utiliser dans votre projet" › "et de l'utiliser pour votre projet" ?

Reply:
Je sais pas trop.
La version anglaise est: "ButterCMS is a headless CMS and blog engine that allows you **to publish structured content to use in your project**.", et la formulation de "et de l'utiliser pour votre projet" me semble bizarre à l'oreille...
Qu'en penses-tu ?

Reply:
La phrase traduite passe aussi, mais en le lisant, c'était un peu gênant. S'il faut, c'est seulement moi. 
Clos le commentaire, et si ça gêne à d'autres personnes, on en discutera plus en profondeur ! 👌🏼

Reply:
Peut-être :
```suggestion
ButterCMS est un CMS headless et un moteur de blog, qui vous permet de publier du contenu structuré, et de l'utiliser dans votre projet.
```
Qu'en penses-tu ?

Reply:
Ah j'approuve ! Ça passe mieux merci!